### PR TITLE
Set parameters directly in airframe config files instead of setting script variables.

### PR DESCRIPTION
**Pull request by mcsauder**
Hi,

This PR begins work in the airframe config files to directly set parameters rather than setting startup script variables.  The `param set` lines are also moved into the `if [AUTOCNF == yes ]` logic blocks, and some alphabetization/ordering of the lists is performed.

You will notice that in `1002_standard_vtol.hil` and `3036_pigeon` there are/were duplicate parameters.  In those instances I have deleted the first occurrence in favor of the last value set.

Additional PRs will follow this PR to continue with the other airframe config files in groups if this PR is accepted.  When this work is complete is will allow a few of the startup script parameters to be deprecated.  

Please let me know if you have any questions or feedback on this PR.

-Mark

### CODE REVIEW

**Comment by mcsauder** [comment]
I would love to fix what went wrong with this build, but it is completely unintelligible. :/

**Comment by dagar** [comment]
Can you rebase on master?

**Comment by mcsauder** [comment]
Thanks @dagar!

**Comment by mcsauder** [comment]
@dagar , @bkueng , what are your thoughts?

File: ROMFS/px4fmu_common/init.d/2100_standard_plane [hunk 1]
```diff
@@ -23,8 +23,8 @@ sh /etc/init.d/rc.fw_defaults
 
 set MIXER AETRFG
 
-set PWM_RATE 50
-set PWM_AUX_RATE 50
+param set PWM_AUX_RATE 50
+param set PWM_RATE 50
```
Comment (bkueng) [inline ROMFS/px4fmu_common/init.d/2100_standard_plane]: This needs to go into autoconf

Comment (mcsauder) [reply ROMFS/px4fmu_common/init.d/2100_standard_plane]: Done.  Thanks!

**Comment by mcsauder** [comment]
Thanks @bkueng.  I've removed the additional spaces in rcS after confirming what @davids5 pointed out, the genromfs file was indeed carrying those extra spaces and not removing them as I had anticipated.  Let me know if I need to address anything else in this PR.  Thanks again for your review!

-Mark

**Pull request merged**